### PR TITLE
Add textfield

### PR DESCRIPTION
**Pull request by nerimartinez**

### CODE REVIEW

File: src/widgets/textfield/widget.spec.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+import View from './widget';
```
Comment (arnaugiralt) [inline src/widgets/textfield/widget.spec.js:1]: Should be `Textfield` instead of `View`

Comment (arnaugiralt) [inline src/widgets/textfield/widget.spec.js:1]: Should be `Textfield` instead of `View`

File: src/widgets/textfield/widget.spec.js [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+      it('calls stop propagation from event', () => {
+        expect(context.e.stopPropagation).toHaveBeenCalled();
+      });
+
+      it('emits input event with localValue', () => {
+        expect(context.$emit).toHaveBeenCalledWith('input', context.localValue);
+      });
+    });
+
+    describe('removeFocus', () => {
```
Comment (arnaugiralt) [inline src/widgets/textfield/widget.spec.js:43]: Should have `#` on the title's property name (`#removeFocus`)

File: src/widgets/textfield/widget.vue [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+  <div
+    :class="[
+      {'text-field_focused': focused},
+      'text-field',
+    ]"
+    @focusin="setFocus"
+    @focusout="removeFocus"
+  >
+    <label for="textfield">{{ label }}</label>
+    <div class="textfield-wrapper">
```
Comment (arnaugiralt) [inline src/widgets/textfield/widget.vue:12]: I'd rename the class to use the dash (`text-field-wrapper`), for consistency, or remove it on other classes. Your call :)

File: src/widgets/textfield/widget.vue [hunk 4]
```diff
@@ -0,0 +1,140 @@
...
+      <div class="text-field__body">
+        <input
+          ref="textfield"
```
Comment (arnaugiralt) [inline src/widgets/textfield/widget.vue:15]: Remove unused ref

**Review by arnaugiralt — CHANGES_REQUESTED** [review]

**Review by arnaugiralt — APPROVED** [review]

**Review by paulinaczybir — COMMENTED** [review]
Could you please add story for this widget? or for some specific reason it was decided to not add it?

**Comment by nerimartinez** [comment]
> Could you please add story for this widget? or for some specific reason it was decided to not add it?

we are gonna implement them later

**Pull request merged**